setup.py

```diff
@@ -166,7 +169,7 @@ def main() -> None:
         url="https://github.com/pytorch/torchdistx",
         license="BSD",
         keywords=["pytorch", "machine learning"],
-        packages=[package_name],
+        packages=[package_name, subpackage_name],
```

Comment:
Why do we need a subpackage for slowMo? It is already under torchdistx right?

Reply:
@cbalioglu  I thought it would be better to arrange all SlowMo relevant code in the `slow_momentum` sub-package. I've included it here, so that `slow_momentum` is picked up during the production build, otherwise it doesn't see all modules under `slow_momentum` folder. Let me know, if it is better to put SlowMo code just under `torchdistx`

Reply:
I see. You might also check out setuptools `find_package()` utility function which would be a cleaner solution.

Reply:
Thanks for a pointer!